school/student.rb

```diff
@@ -1,10 +1,5 @@
 require_relative 'person'
 class Student < Person
-  def initialize(classroom)
-    @classroom = classroom
-    super()
-  end
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly previous reviewer mentioned about your `def`, please don't remove this part so constructor extends parent's constructor by adding `@classroom` and a `parameter` for it like that🙏🏼 

```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```